src/styles/atoms/_labels.scss

```diff
@@ -0,0 +1,7 @@
+.label{
+  color: $color-grey-lighter;
+  padding: 5px;
+  border-top-right-radius: 5px;
+  border-top-left-radius: 5px;
```

Comment:
Wanted to match the radius with the ones on the input fields, but fair point sire!